plugin.xml

```diff
@@ -1,50 +1,48 @@
 <root>
-   <name>FormCreator</name>
+   <name>Formcreator End-of-Life Updater</name>
    <key>formcreator</key>
    <state>stable</state>
    <logo>https://raw.githubusercontent.com/pluginsGLPI/formcreator/master/icon.png</logo>
    <description>
       <short>
-         <en>
-            Transition plugin marking the end of life of Formcreator.
-            It updates database tables and data to the final version, required before the automatic migration of forms into GLPI 11 core.
-            ⚠️ This plugin must be installed in production only to perform the update, and then uninstalled once the operation is complete.
-         </en>
+         <fr>Plugin marquant la fin de vie du plugin FormCreator.
+             Il met à jour les tables et les données de la base vers la version finale, prérequis avant la migration automatique des formulaires vers les assets personnalisés de GLPI 11.
+             ⚠️ Ce plugin doit être installé uniquement en production pour effectuer la mise à jour, puis désinstallé une fois l’opération terminée.</fr>
+         <en>Plugin marking the end of life of FormCreator plugin.
+             It updates database tables and data to the final version, prerequisites before the automatic migration of forms into GLPI 11 custom assets.
+             ⚠️ This plugin must be installed in production only to perform the update, and then uninstalled once the operation is complete.</en>
       </short>
       <long>
-         <en>
-Formcreator v3.0.0 - Migration Only Plugin (End of Life)
+	      <fr>
+FormCreator 3.0.0 - Plugin de migration uniquement (Fin de vie)
 
-IMPORTANT NOTICE: This version (3.0.0) is a migration-only plugin designed exclusively for GLPI 11.0+.
+Si vous prévoyez de migrer votre instance GLPI vers la version 11 et que vous n'avez pas installé la version 2.14.14 sur votre instance GLPI 10.0.x, une version spécifique compatible avec GLPI 11 est disponible.
```

Comment:
i guess you meant the unreleased version 2.14.0 ?

Reply:
No, I was referring to the previous official GLPI version, 10.0.x.

fixed